### PR TITLE
Adding conf dict to context

### PR DESCRIPTION
**Pull request by mistercrunch**
as a solution to https://github.com/airbnb/airflow/issues/168

### CODE REVIEW

**Comment by mistercrunch** [comment]
Passed the tests, mergin'

**Comment by r39132** [comment]
Hi @mistercrunch !
I looked at the diff. I didn't see any material change. Just some formatting changes and a documentation change. Excuse me if I missed something -- it might also be pretty late at night!

-s

File: airflow/models.py [hunk 1]
```diff
@@ -934,7 +934,8 @@ def get_template_context(self):
             'task': task,
             'task_instance': self,
             'ti': self,
-            'task_instance_key_str': ti_key_str
+            'task_instance_key_str': ti_key_str,
+            'conf': conf,
```
Comment (mistercrunch) [inline airflow/models.py:938]: This line right here

**Comment by r39132** [comment]
Ah... makes sense.

**Pull request merged**